src/components/map/MapboxMap.tsx

```diff
@@ -0,0 +1,209 @@
+import ReactDOMServer from "react-dom/server";
+import React, { useContext, useEffect, useRef } from "react";
+import { AppContext } from "context";
+import { EstimateGradePrevalence, LanguageType } from "types";
+import mapboxgl from "mapbox-gl";
+import httpClient from "httpClient";
+import { getEsriVectorSourceStyle, addEsriLayersFromVectorSourceStyle } from "components/map/EsriMappingUtil";
+import generateSourceFromRecords from "./GeoJsonGenerator";
+import StudyPopup from "components/map/StudyPopup";
+import CountryPopup from 'components/map/CountryPopup'
+import "components/map/Map.css";
+import "components/map/MapboxMap.css";
+import "mapbox-gl/dist/mapbox-gl.css";
+
+mapboxgl.accessToken = "pk.eyJ1Ijoic2Vyb3RyYWNrZXIiLCJhIjoiY2tha2d4bTdmMDJ3dzJ3azFqbnphdWlzZSJ9.IutISibpBV33t_7ybaCNTg";
```

Comment:
I think we should make this an env var. Esp if we're paying for mapbox, this is not good to be committing in a public repo.

Reply:
oops, I missed this. I'll refresh the token anyways when we deploy, but def need to add that to the env